.gitlab-ci.yml

```diff
@@ -886,6 +889,7 @@ workflow:
   - changes:
       paths:
         # TODO: Add paths that should trigger tests for npm
+        - pkg/network/**/*
```

Comment:
is this unrelated?

Reply:
Actually not really related to the PR, but looks like we added a linter after the rule was created that force me to add at least one path there.